### ISSUE TITLE
Overwrite or merge

### ISSUE THREAD

**Issue by purplecabbage** [adobe/aio-cli-plugin-app#262]
**Describe the bug**
Users are faced with difficult decisions before even looking at their project

**To Reproduce**
1. `aio app init -y`
1. add more than 1 action when given the choice

After the second action is added, you are confronted with the stressful choice of overwriting or merging ...

```
project/.env already exists: (omxH)
project/.aio already exists (omxH)
```

... at this point a user has zero context to make this decision, they have never even seen these files so how would they know which choice is right?

**Expected behavior**
The initialization of an app should do the right things based on the options the user has selected, they should not be asked to make difficult choices.

**Comment by meryllblanchet** [comment]
+1. It happened to me this week and I wondered why I had to take this decision while I was bootstrapping a plain new project.

**Comment by aiojbot** [comment]
JIRA issue created: https://jira.corp.adobe.com/browse/ACNA-814

**Comment by Himavanth** [comment]
The execution of second command `aio app add action` has no idea if the files were changed after `init`.
Hence the ask for confirmation on file conflict. Seems fair.
Did i understand this right ? @purplecabbage @meryllblanchet

**Comment by moritzraho** [comment]
what about always merging ? this is the less destructive option

**Comment by sandeep-paliwal** [comment]
Tried `aio app init` with two actions selected. The init went fine without asking for any file overwrite. As per issue description app init asks confirmation in such case. Couldn't repro this .
After init I ran `aio app add action` this asked for user confirmation for overwriting `manifest.yml` .
Do we want to merge files for such cases and avoid asking users any confirmation? @meryllblanchet @sarahxxu
